apps/performance-tests/src/util/Datasets.ts

```diff
@@ -127,4 +164,203 @@ export class Datasets {
 
     console.log(`${numElements} elements: Done.`);
   }
+
+  private static async createNestedElementModel(name: string, localPath: string, numElements: number) {
```

Comment:
Please add a comment next to each of these `create***IModel` functions explaining the IModel content. For example:

```
Create an iModel with `numElements` elements, half of them 2d and the other half - 3d, all belonging to the same 2d
and 3d model and category. The elements are set up in hierarchical manner, with 1000 top level elements each having 1
child element, which has 1 child element, and so on until the depth of 25 elements is reached.
```

Reply:
Added comments.